cwms-data-api/src/main/java/cwms/cda/data/dto/texttimeseries/TextTimeSeries.java

```diff
@@ -0,0 +1,275 @@
+package cwms.cda.data.dto.texttimeseries;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.data.dto.CwmsDTO;
+import hec.data.timeSeriesText.DateDateComparator;
+import hec.data.timeSeriesText.DateDateKey;
+import hec.data.timeSeriesText.TextTimeSeriesRow;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.NavigableMap;
+import java.util.Set;
+import java.util.TreeMap;
+
+
+@JsonDeserialize(builder = TextTimeSeries.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+public class TextTimeSeries extends CwmsDTO {
```

Comment:
I recommend adding the location time-zone and the interval offset similar to numerical time series for LRTS.

Reply:
I added them to the DTO but its gonna take me a bit to figure out how to query for them.  I know the timeseries has a metadata query that fills some of that in...

Reply:
The time-zone and interval offset will be addressed by https://github.com/USACE/cwms-data-api/issues/547